packages/flutter/test/rendering/object_test.dart

```diff
@@ -4,10 +4,14 @@
 
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter_test/src/binding.dart' show TestWidgetsFlutterBinding;
 import '../flutter_test_alternative.dart';
```

Comment:
if we import flutter_test here we shouldn't import flutter_test_alternative. We should pick one or the other.